apps/web/src/pages/Applications/ApplicationProfilePage/components/DiversityEquityInclusion/DiversityEquityInclusion.tsx

```diff
@@ -60,6 +66,19 @@ const DiversityEquityInclusion = ({
       >
         {intl.formatMessage(title)}
       </ToggleSection.Header>
+      {isIAP && !isComplete && (
+        <Well color="error">
+          <p>
+            {intl.formatMessage({
+              defaultMessage:
+                "This opportunity is reserved for Indigenous candidates.",
+              id: "+M9Pwt",
+              description:
+                "Error message displayed when a user's equity information does not match an opportunity",
+            })}
```

Comment:
May be worth moving this to `~/messages/applicationMessages` to keep both instances in sync easier.

Reply:
Sure, done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6579/commits/19c4298824b6f99400f42905d1e09d37d7f8c8a8